config/clock.rb

```diff
@@ -48,6 +48,8 @@ class Clock
     TeacherTrainingPublicAPI::SyncAllProvidersAndCoursesWorker.perform_async(false)
   end
 
+  every(7.days, 'TADDegreeClassExport', at: 'Sunday 23:59') { SupportInterface::TADDegreeClassExport.run_weekly }
```

Comment:
I've got concerns generally with these reports about what happens if a report fails. If we have to fix a problem and end up re-running it on say Monday afternoon then it's not going to be an entirely accurate picture of the start of the week. (But that's not something specific or that needs addressing in this PR).